controllers/session.go

```diff
@@ -0,0 +1,68 @@
+package controllers
+
+import (
+	"go-google-scraper-challenge/forms"
```

Comment:
How about grouping the import?

Reply:
@gutakk done on [3082c3d](https://github.com/carryall/go-google-scraper-challenge/pull/31/commits/3082c3d64a79f029d983365d2d8461b6ecfedb3d) along with the `form`